custom/conf/app.ini.sample

```diff
@@ -299,7 +299,12 @@ ENABLE_NOTIFY_MAIL = false
 ENABLE_REVERSE_PROXY_AUTHENTICATION = false
 ENABLE_REVERSE_PROXY_AUTO_REGISTRATION = false
 ; Enable captcha validation for registration
-ENABLE_CAPTCHA = true
+ENABLE_CAPTCHA = false
```

Comment:
Do not change default as this way it will be breaking change

Reply:
The default value is actually `false` but the documentation is wrong

Reply:
Hi. Yes `false` is the default, and I changed it to be correct like @JonasFranzDEV said.

Reply:
I would like you use a captcha type and currently you have three options: disabled, image and recaptcha and default is disabled.